<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SafeGuard</class>
 <widget class="QWidget" name="SafeGuard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>1018</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SafeGuard</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>信号连接：</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>设备列表：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="list"/>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_scan">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string> 扫描设备</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QCheckBox" name="checkBox_discoverable">
          <property name="maximumSize">
           <size>
            <width>800</width>
            <height>400</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>  蓝牙可见</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton_openBluetooth">
          <property name="text">
           <string>打开设备</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_disconnect">
          <property name="text">
           <string>断开连接</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_closeDevice">
          <property name="text">
           <string>关闭设备</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>提示：请确定链接正确的设备！</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>运行状态：</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>状态指示：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>报警状态：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_stateText">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>待机中....</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButton_alartLed">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="textBrowser"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>图像回传区域：</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="imageLabel">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>400</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>400</height>
         </size>
        </property>
        <property name="text">
         <string>图像区域</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>控制区域：</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_warning">
        <property name="text">
         <string>语音喊话</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_readImage">
        <property name="text">
         <string>读取图像</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_id">
        <property name="text">
         <string>身份认证</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_clear">
        <property name="text">
         <string>清除报警状态</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pushButton_openBluetooth</tabstop>
  <tabstop>checkBox_discoverable</tabstop>
  <tabstop>list</tabstop>
  <tabstop>pushButton_alartLed</tabstop>
  <tabstop>pushButton_scan</tabstop>
  <tabstop>pushButton_warning</tabstop>
  <tabstop>pushButton_readImage</tabstop>
  <tabstop>pushButton_id</tabstop>
  <tabstop>pushButton_clear</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
